revolv/static/project.scss

```diff
@@ -77,3 +476,227 @@
+    }
+
+    .dashboard {
+        height: 700px;
+        z-index: 1;
+        top: 0px;
+
+        .statistics-graphics-container {
+            height: 500px;
+            .bar {
+                height: 25px;
+
+                &.top-bar {
+                    top: 63%;
+                }
+                &.bottom-bar {
+                    top: 68%;
+                }
+            }
+
+            .container {
+                .statistic {
+
+                    &.left-text {
+                        float: left;
+                        height: 100%;
+                        margin: 0 0 0 10px;
+                        font-size: 0.8rem;
+                        line-height: 25px;
+
+                    }
+
+                    &.right-text {
+                        float: right;
+                        height: 100%;
+                        margin: 0 10px 0 0;
+                        font-size: 0.8rem;
+                        line-height: 25px;
+
+                    }
+                }
+            }
+        }
+    }
+
+    .header-img-div {
+        height: 270px;
+
+        .overlay {
+            
+            .donate-button-header {
+                left: 15%;
+                width: 70%;
+                height: 15%;
+                top: 170%;
+                border-color: $revolv-dark-blue;
+                // width: 0;
+                // height: 0;
+                z-index: 1000;
+
+                .donate-link {
+                    line-height: 40px;
+                    margin-top: 0;
+
+                    p {
+                        height: 100%;
+                        line-height: 40px;
+                        color: $revolv-dark-blue;
+                    }
+                }
+            }
+
+            .project-title {
+                font-size: 1.5rem;
+                line-height: 1.5rem;
+                // top: 20px;
+                // width: 90%;
+                // left: 5%;
+
+                top: 140px;
+                width: 100%;
+                text-align: center;
+                left: 0;
+            }
+        }
+    }
+
+
+
+    .about {
+
+        margin-top: 20px;
+        .header-text {
+            p {
+                font-size: 1.5rem;
+                line-height: 1.5rem;
+            }
+        }
+
+        .categories {
+
+            .category-holder {
+                width: 25%;
+            }
+        }
+
+        .about-text {
+            p {
+                font-size: 1rem;
+            }
+        }
+
+    }
+
+    .map {
+        .title-container {
+            p {
+                font-size: 1rem;
+                line-height: 1rem;
+            }
+        }
+
+        .address-container {
+            p {
+                font-size: 0.8rem;
+                line-height: 0.8rem;
+            }
+        }
+
+        .donate-button-header {
+            p {
+                font-size: 1.5rem;
+            }
+        }
+    }
+}
+
+@media #{$medium-only} {
+    .dashboard {
```

Comment:
It's probably because of how [CSS Specificity](http://www.smashingmagazine.com/2007/07/27/css-specificity-things-you-should-know/) works. Should be fine for now.